app/views/application/_card.html.haml

```diff
@@ -9,6 +9,7 @@
         = image_tag account.avatar.url, alt: '', width: 48, height: 48, class: 'u-photo'
 
       .display-name
+        %span{:id=>"default_account_display_name", :style=>"display:none;"}= display_name(account, custom_emojify: true)
```

Comment:
Use `id: ""` style, the `=>` style is outdated

Reply:
since the default display name is always just the username, we should display that instead. this will give incorrect results for users trying to unset their display name